ClassLibrary/ClassLibrary.Test/TestDepartmentReaders.cs

```diff
@@ -82,5 +82,48 @@ public void BorrowedBookNullArgument()
             Should.Throw<ArgumentNullException>(() => DR.BorrowBook(reader, null));
         }
 
+        [Test]
+        public void NotList_GetTimeWhenFreeBook()
+        {
+            DepartmentReaders DR = new DepartmentReaders();
+            //I think it doesn't matter which parameter returns. What do you think? 
```

Comment:
you're right, there's no "right" answer. but you should be able to make a grounded choice, so think what are pros and cons of each option.

Reply:
+